lib/modules/ens/contracts/ENSRegistry.sol

```diff
@@ -15,8 +15,8 @@ contract ENSRegistry is ENS {
     mapping (bytes32 => Record) records;
 
     // Permits modifications only by the owner of the specified node.
-    modifier only_owner(bytes32 node) {
-        require(records[node].owner == msg.sender);
+    modifier only_owner(bytes32 node, address owner) {
```

Comment:
Why are you modifying an ENS contract?
Those contract are already deployed on main net no?
So if you change them, dev and live won't behave the same